.storybook/css/blaze-ui.css

```diff
@@ -5352,6 +5356,111 @@ hr {
   border-color: #00bcd4;
 }
 
+.custom-DraftEditor-editor .public-DraftEditorPlaceholder-root,
+.custom-DraftEditor-editor .public-DraftEditor-content {
+  padding: 10px;
+  margin: 0 -10px -10px;
+}
+
+.custom-DraftEditor-editor .public-DraftEditor-content {
+  min-height: 100px;
+}
+
+.custom-DraftEditor-editor .public-DraftEditorPlaceholder-inner {
+  color: #cccccc;
+}
+
+.custom-DraftEditor-hidePlaceholder .public-DraftEditorPlaceholder-root {
+  display: none;
+}
+
+.custom-DraftEditor-editor .public-DraftStyleDefault-pre {
+  background-color: white;
+  border: 0;
+}
+
+.custom-DraftEditor-root {
+  background: #ffffff;
```

Comment:
I think we are using constants for all colours.

Reply:
is the compiled file

Reply:
Oh sorry, didn't realise this is the library and we manage the styles separately. My bad.